### DISCUSSION TITLE
temux-github

### DISCUSSION THREAD

**Discussion by jaiyid1986** [community/community#68280] [New to GitHub]
bagaimna caranya akun github bisa d simpan d termux

**Comment by queenofcorgis** [comment]
Hi @jaiyid1986 , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
